lib/src/Pos/Lrc/Worker.hs

```diff
@@ -196,10 +198,10 @@ issuersComputationDo epochId = do
 
 leadersComputationDo :: LrcMode ssc ctx m => EpochIndex -> SharedSeed -> m ()
 leadersComputationDo epochId seed =
-    unlessM (isJust <$> getLeaders epochId) $ do
+    unlessM (isJust <$> LrcDB.getLeadersForEpoch epochId) $ do
```

Comment:
Here and above it makes more sense to use `getLeader (SlotId epochId minBound)`. It should be `Just` iff `LrcDB.getLeadersForEpoch epochId` is `Just`.